packages/ui-extensions-go-cli/api/api.go

```diff
@@ -127,14 +130,15 @@ func getJSONRawMessage(data interface{}) (result json.RawMessage, err error) {
 	return
 }
 
-func interfaceToMap(data interface{}) (result map[string]interface{}, err error) {
+func interfaceToMap(data interface{}, logBuilder logging.LogEntryBuilder) (result map[string]interface{}, err error) {
```

Comment:
Do this kind of utility methods knows about logEntryBuilder?, maybe different error types could be returned so the caller can logged properly. This suggestion could avoid adding new parameters to the methods